assets/app/scripts/controllers/projects.js

```diff
@@ -8,13 +8,19 @@
  * Controller of the openshiftConsole
  */
 angular.module('openshiftConsole')
-  .controller('ProjectsController', function ($scope, $location, DataService) {   
+  .controller('ProjectsController', function ($scope, $rootScope, $location, DataService, AuthService) {   
     $scope.projects = {};
+    $scope.alerts = $scope.alerts || {};
+    $scope.emptyMessage = "Loading...";
 
-    DataService.list("projects", $scope, function(projects) {
-      $scope.projects = projects.by("metadata.name");
+    AuthService.withUser().then(function(user) {
+      $rootScope.user = user;
+      DataService.list("projects", $scope, function(projects) {
+        $scope.projects = projects.by("metadata.name");
+        $scope.emptyMessage = "You have no projects. For an example, run <code>openshift cli create -f examples/hello-openshift/hello-project.json</code>, then refresh this page";
```

Comment:
Should probably point people to the example under examples/sample-app/project.json